index.html

```diff
@@ -1,13 +1,146 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
     <title>Full Stack Portfolio</title>
   </head>
   <body>
-
+    <header>
+      <div class="logo-container">
+        <a href="/" class="logo">Thomas Jensen</a>
+        <a href="/" class="logo-heb">תומס ינסן</a>
+      </div>
+      <div class="links">
+        <a href="#about-me">About Me</a>
+        <a href="#portfolio">Portfolio</a>
+        <a href="#experience">Experience</a>
+      </div>
+    </header>
+    <main>
+      <div class="splash">
+        <br />
```

Comment:
maybe use a margin instead of br? a <div> with a specific class that adds a margin as required